python/cxx/deck_keyword.cpp

```diff
@@ -56,11 +59,70 @@ struct DeckRecordIterator
     }
 };
 
+
+void push_string_as_deck_value(std::vector<DeckValue>& record, const std::string str) {
+
+    if (str.back() == '*') {
```

Comment:
The current code does not support `5*$value` which should amount to a five times repeat of `$value` - would maybe be simpler to move the `*` handling to the main function (which is just a lambda now - and could be upgraded to a proper function). Something like this:

```
if (string_arg.find('*') != std::string::npos) {
   std::size_t star_pos = string_arg.find('*');
   int multiplier = 1;
   std::string string_value = "";
   try {
        multiplier = std::stoi(string_value.substr(0, star_pos));
   } catch (std::invalid_argument& e) { };

   if (star_pos < string_arg.size())
       string_value = string_arg.substr(star_pos + 1);
...
```

Or something like that. Sorry this task has been a bit back and forth - we'll soon merge.

PS: You could look at: https://github.com/OPM/opm-common/blob/c7497497f671bed3bbfcf024b14a95da6527f6c9/src/opm/parser/eclipse/Parser/raw/StarToken.cpp - that is directly not directly usable, but inspiration.